scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -17,20 +17,20 @@ package com.twitter.scalding
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.mapred.JobConf
-import org.apache.hadoop.io.serializer.{ Serialization => HSerialization }
-import com.twitter.chill.{ ExternalizerCodec, ExternalizerInjection, Externalizer, KryoInstantiator }
-import com.twitter.chill.config.{ ScalaMapConfig, ConfiguredInstantiator }
-import com.twitter.bijection.{ Base64String, Injection }
-
+import org.apache.hadoop.io.serializer.{Serialization => HSerialization}
```

Comment:
if you could revert the style changes here that'd be great. I think scalafmt will put them back anyway.